adc/ad7779/imxrt/dma.c

```diff
@@ -41,8 +41,12 @@ static struct {
 static int edma_error_handler(unsigned int n, void *arg)
 {
 	/* TODO: Store some info for debugging? Notify about it somehow? */
-	sai_rx_disable();
-	edma_clear_error(SAI1_RX_DMA_CHANNEL);
+	uint32_t mask = 1U << SAI1_RX_DMA_CHANNEL;
```

Comment:
nit: maybe `const uint32_t mask` or `static const`, we use `-Os` on `imxrt` or at least `-O1` so it's optimized to `const`, but when debugging is enabled it is allocated on stack.